content/en/serverless/step_functions/merge-step-functions-lambda.md

```diff
@@ -178,9 +178,49 @@ To link your Step Function traces to nested Step Function traces, configure your
 }
 {{< /highlight >}}
 
-## Merge Lambda -> Step Functions -> Lambda
+## Merge Any Upstream Service -> Step Functions -> Lambda
```

Comment:
Open to a better way to phrase these two sections. For context here are some example cases:
* Lambda -> Step Functions -> Lambda
* Lambda -> Step Functions -> Step Functions
* Step Functions -> Step Functions -> Step Functions
* Step Functions -> Step Functions -> Lambda

The setup is *slightly* different depending on if the downstream is a Lambda or Step Function but I wasn't sure the best way to keep it as one section with a split for the value for the `Arguments` key. I think one section would make sense as both sections have the same `JSONata` requirement that's currently duplicated.